### PR TITLE
[IMP] edi: manage the case when no check is defined

### PR DESCRIPTION
**Pull request by etobella**
When no check is defined, we could assume that it was processed properly. WDYT?

@simahawk

### CODE REVIEW

**Comment by simahawk** [comment]
while it might be good not to raise an error, IDT is a good idea to change the state.
You should handle the transition in your own implementation probably (eg: automated action).
Can you elaborate a bit on the use case?

**Comment by etobella** [comment]
Well, with your comment, I have to implement every time a check that everything worked fine, but it is not already correct. For example, if you use a Webservice, you know at the end that everything went right, but you cannot change the information on the component, as the state will be written later. Without the state change we cannot expect to use queue in this cases. Also, why using a new automated action if we already have this one?

**Comment by simahawk** [comment]
Sorry, I still don't get the point. Seems you are mixing up "webservice" case and "storage" case. We should probably have a chat around this :)

**Comment by etobella** [comment]
@simahawk Changes applied as talked

File: edi/models/edi_backend.py [hunk 1]
```diff
@@ -37,6 +37,7 @@ class EDIBackend(models.Model):
         required=True,
         ondelete="restrict",
     )
+    output_sent_processed_auto = fields.Boolean()
```
Comment (simahawk) [inline edi/models/edi_backend.py]:
```suggestion
    output_sent_processed_auto = fields.Boolean(help=""""
    Automatically set the record as processed after sending.
    Usecase: the web service you send the file to processes it on the fly.
    """)
```

**Review by simahawk — APPROVED** [review]

**Review by olgamarcocb — APPROVED** [review]
:+1:

**Review by albariera — APPROVED** [review]

**Pull request merged**